spec/statesman/adapters/active_record_spec.rb

```diff
@@ -163,6 +163,39 @@
             to(change { previous_transition.reload.updated_at })
         end
 
+        context "with a custom updated timestamp column set" do
+          before do
+            RemoveUpdatedAtFromMyActiveRecordModelTransitions.migrate(:up)
+            AddUpdatedOnToMyActiveRecordModelTransitions.migrate(:up)
+            MyActiveRecordModelTransition.updated_timestamp_column = :updated_on
+            MyActiveRecordModelTransition.reset_column_information
+          end
+
+          after do
+            MyActiveRecordModelTransition.updated_timestamp_column =
+              MyActiveRecordModelTransition::DEFAULT_UPDATED_TIMESTAMP_COLUMN
+          end
+
+          it "touches the previous transition's updated_on timestamp" do
+            expect { Timecop.freeze(Time.now + 5.seconds) { create } }.
+              to(change { previous_transition.reload.updated_on })
+          end
+        end
+
+        context "with no updated timestamp column set" do
+          before { MyActiveRecordModelTransition.updated_timestamp_column = nil }
+
+          after do
+            MyActiveRecordModelTransition.updated_timestamp_column =
+              MyActiveRecordModelTransition::DEFAULT_UPDATED_TIMESTAMP_COLUMN
+          end
```

Comment:
Think you should be able to do
```ruby
around do |example|
  MyActiveRecordModelTransition.updated_timestamp_column.tap do |original|
    MyActiveRecordModelTransition.updated_timestamp_column = nil
    example.run
    MyActiveRecordModelTransition.updated_timestamp_column = original
end
```

Unsure if that's any neater though...

Reply:
I think that's nicer as it clearly relates the before and after actions. Will do that now.

Reply:
I haven't managed to do this. It's working for the `"with no updated timestamp column set"` context (which doesn't use migrations) but not for the `"with a custom updated timestamp column set"` context. 

The issue I'm seeing is that Rails thinks the column has changed (even when you call `.reset_column_information`), but SQLite complaints that the column doesn't exist. I'm going to try a different approach of creating the `updated_on` column by default, and only using it in some tests.